packages/common/src/modules/register.ts

```diff
@@ -63,7 +60,6 @@ container.bind(EpgController).toSelf();
 // Integration controllers
 container.bind(AccountController).toSelf();
 container.bind(CheckoutController).toSelf();
-container.bind(ProfileController).toSelf();
```

Comment:
I think it won't work as we still use hooks based on the controller (not ideal).